### ISSUE TITLE
Problems with audio models

### ISSUE THREAD

**Issue by Nerual-Dreming** [diStyApps/VisionCrafter#21]
Hi, first of all I want to say that this software is just great in spite of the early version. I am absolutely delighted. 
But I have encountered this problem, when I add music to my generation I get an error.

`OSError: Not found: "C:\Users\Илья/.cache\huggingface\hub\models--t5-base\snapshots\fe6d9bf207cd3337512ca838a8b453f87a9178ef\spiece.model": No such file or directory Error #2`

As I understand it, this error is caused by the fact that I have Cyrillic in the username path, but the problem is more likely that the audio generator module stores its files in a different path, not inside the venv as would be reasonable. 

please make it so that all module files are stored inside the installation directory, I'm sure this will solve a lot of problems in the future.

**Comment by AsaTyr2018** [comment]
i also found out that the current version installs by default in the user dir. 
special the Huggingface cache.

All i found is this references:
https://huggingface.co/docs/datasets/cache

did you tried to set a custom project path?
![grafik](https://github.com/diStyApps/VisionCrafter/assets/43162495/68b92111-df9f-4c7a-afd8-dcb1140fa84a)


Maby this helps.

**Comment by diStyApps** [comment]
> Hi, first of all I want to say that this software is just great in spite of the early version. I am absolutely delighted. But I have encountered this problem, when I add music to my generation I get an error.
> 
> `OSError: Not found: "C:\Users\Илья/.cache\huggingface\hub\models--t5-base\snapshots\fe6d9bf207cd3337512ca838a8b453f87a9178ef\spiece.model": No such file or directory Error #2`
> 
> As I understand it, this error is caused by the fact that I have Cyrillic in the username path, but the problem is more likely that the audio generator module stores its files in a different path, not inside the venv as would be reasonable.
> 
> please make it so that all module files are stored inside the installation directory, I'm sure this will solve a lot of problems in the future.

Yes, i don't like it too, ill address it soon so they will be inside the installation directory.
